src/poliastro/twobody/propagation.py

```diff
@@ -88,24 +88,23 @@ def cowell(k, r, v, tofs, rtol=1e-11, *, events=None, ad=None, **ad_kwargs):
     tofs = tofs.to(u.s).value
 
     u0 = np.array([x, y, z, vx, vy, vz])
+    t0 = 0
 
     # Set the non Keplerian acceleration
     if ad is None:
 
         def ad(t0, u_, k_):
             return 0, 0, 0
 
-    f_with_ad = functools.partial(func_twobody, k=k, ad=ad, ad_kwargs=ad_kwargs)
+    p = [k, ad, ad_kwargs]
 
-    result = solve_ivp(
-        f_with_ad,
-        (0, max(tofs)),
+    result_solver = nbkode.DOP853(
+        func_twobody,
+        t0,                     
         u0,
-        rtol=rtol,
-        atol=1e-12,
-        method=DOP853,
-        dense_output=True,
-        events=events,
+        params = p,
+        rtol = rtol,
+        atol = 1e-12,
     )
     if not result.success:
```

Comment:
Should be `result_solver` rather than `result`

Reply:
This is invalid for the current implementation... will update in the next commit